addons/base_import_module/models/ir_module.py

```diff
@@ -141,11 +141,13 @@ def _import_module(self, module, path, force=False, with_demo=False):
                         convert_xml_import(self.env, module, fp, idref, mode, noupdate)
                         if filename in exclude_list:
                             self.env['ir.model.data'].create([{
-                                'name': f"cloc_exclude_{key}",
+                                'name': f"{module}_{key}",
```

Comment:
is it a good idea to change that, what will happen if the module was previously installed with cloc_exclude name ?